diff --git a/node_modules/bip39/index.js b/node_modules/bip39/index.js
index efed68c..927afc8 100644
--- a/node_modules/bip39/index.js
+++ b/node_modules/bip39/index.js
@@ -3,7 +3,7 @@ var createHash = require('create-hash')
 var _pbkdf2 = require('pbkdf2')
 var pbkdf2 = _pbkdf2.pbkdf2Sync
 var pbkdf2Async = _pbkdf2.pbkdf2
-var randomBytes = require('randombytes')
+var {randomBytes} = require('react-native-crypto')
 
 // use unorm until String.prototype.normalize gets better browser support
 var unorm = require('unorm')
