diff --git a/node_modules/react-native-radial-gradient/ios/SRSRadialGradient.podspec b/node_modules/react-native-radial-gradient/SRSRadialGradient.podspec
similarity index 89%
rename from node_modules/react-native-radial-gradient/ios/SRSRadialGradient.podspec
rename to node_modules/react-native-radial-gradient/SRSRadialGradient.podspec
index 008320d..506db86 100644
--- a/node_modules/react-native-radial-gradient/ios/SRSRadialGradient.podspec
+++ b/node_modules/react-native-radial-gradient/SRSRadialGradient.podspec
@@ -1,5 +1,5 @@
 require 'json'
-version = JSON.parse(File.read('../package.json'))["version"]
+version = JSON.parse(File.read('./package.json'))["version"]
 
 Pod::Spec.new do |s|
 
